filebeat/fileset/modules.go

```diff
@@ -285,6 +294,40 @@ func (reg *ModuleRegistry) LoadPipelines(esClient PipelineLoader) error {
 	return nil
 }
 
+func scriptPayload(source, lang string) map[string]interface{} {
+	s := make(map[string]string)
+	s["lang"] = lang
+	s["source"] = source
+	payload := make(map[string]interface{})
```

Comment:
I suggest we use `common.MapStr` here mainly because we use it everywhere else where we have to create json. It has the same base interface.